.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,67 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ unstable ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ unstable ]
+  schedule:
+    - cron: '19 5 * * 0'
```

Comment:
is there really any reason to run this weekly?
how about push / PR to release branches?
maybe it should just be? (like the CI):
```
on: [push, pull_request]
```

Reply:
the weekly is just to make sure that if a new vulnerability was added to the the Database the code will be checked even if no PR/push was made.